gita/utils.py

```diff
@@ -121,14 +122,22 @@ def add_repos(repos: Dict[str, str], new_paths: List[str]):
         start_new_session=True,
         cwd=path)
     stdout, stderr = await process.communicate()
-    stdout and print(stdout.decode())
-    stderr and print(stderr.decode())
+    if stdout:
+        stdout = stdout.decode()
+        print(format_output(stdout, path))
+    if stderr:
+        stderr = stderr.decode()
+        print(format_output(stderr, path))
     # The existence of stderr is not good indicator since git sometimes write
     # to stderr even if the execution is successful, e.g. git fetch
     if process.returncode != 0:
         return path
 
 
+def format_output(output: str, prefix: str):
+    return '\n'.join([prefix + ': ' + line for line in output.split('\n')])
```

Comment:
please use `f'{prefix}: {line}' for line in output.split('\n')`

f-string is more efficient. See
https://stackoverflow.com/questions/38722105/format-strings-vs-concatenation

Reply:
fixed in next commit